content/99.back-matter.md

```diff
@@ -3,13 +3,13 @@
 We would like to thank the authors of the Deep Review who helped us test collaborative writing with Manubot.
 The authors who responded favorably to being acknowledged are Paul-Michael Agapow, Amr M. Alexandari, Brett K. Beaulieu-Jones, Anne E. Carpenter, Travers Ching, Evan M. Cofer, Dave DeCaprio, Brian T. Do, Enrico Ferrero, David J. Harris, Michael M. Hoffman, Alexandr A. Kalinin, Anshul Kundaje, Jack Lanchantin, Christopher A. Lavender, Benjamin J. Lengerich, Zhiyong Lu, Yifan Peng, Yanjun Qi, Gail L. Rosen, Avanti Shrikumar, Srinivas C. Turaga, Gregory P. Way, Laura K. Wiley, Stephen Woloszynek, Wei Xie, Jinbo Xu, and Michael Zietz.
 In addition, we thank Ogun Adebali, Evan M. Cofer, and Robert Gieseke for contributing to the Manubot template manuscript.
-We are grateful for additional Manubot discussion and testing by Alexander Dunkel, Ansel Halliburton, Achintya Rao, and other GitHub users.
-Setup and maintenance of the Zotero translation-server for Manubot usage was performed by Dongbo Hu.
-We thank John MacFarlane for assistance with pandoc and the global Binder team for advice on Binder.
+We are grateful for additional Manubot discussion and testing by Alexander Dunkel, Ansel Halliburton, Benjamin J. Heil, Zach Hensel, Alexandra J. Lee, YoSon Park, Achintya Rao, and other GitHub users.
+We thank John MacFarlane and Nikolay Yakimov for assistance with Pandoc and the global Binder team for advice on Binder.
+Finally, we thank C. Titus Brown and the other anonymous reviewers for their help improving this manuscript.
```

Comment:
I think somewhere we should mention that the peer review history is available in `response-to-reviewers`. Open peer review for journals that do not support it is another benefit of Manubot.

Reply:
Do you suggest directly linking to that document?  If so, we can save that for the end so that we link to the final version.

I can add a sentence like

> Their feedback and our responses are available [on GitHub]().

Reply:
> Do you suggest directly linking to that document? If so, we can save that for the end so that we link to the final version.

Yes.

Reply:
Added in a864a89 with a **TODO** to update the URL.